RELEASING.md

```diff
@@ -2,7 +2,9 @@
 
 - Make a branch `bump/x.x.x`
 - Update purchases-android version in `android/common/build.gradle`
-- Update CHANGELOG.md
+- Update purchases-ios pod version in `PurchasesHybridCommon.podspec`, `ios/PurchasesHybridCommon/Podfile`, and `ios/PurchasesHybridCommon/PurchasdsHybridCommon.xcworkspace` (open this last one with Xcode)
+- Update `CHANGELOG.md`
```

Comment:
we'll eventually wanna automate this, but my thinking was to first have it working and in use

Reply:
Yeah this works